modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -347,6 +347,10 @@ bool QueueReader::getInputVector(std::vector<Q*> &in_queues,
                                  cv::GRunArgs    &in_constants,
                                  cv::GRunArgs    &isl_inputs)
 {
+    // NB: Need to release resources from the previous step, to fetch new ones.
+    // On some systems it might be impossible to allocate new memory
+    // until the old one is released.
+    m_cmd.clear();
```

Comment:
what about the `isl_inputs` below?

Reply:
It's already empty: https://github.com/opencv/opencv/blob/master/modules/gapi/src/executor/gstreamingexecutor.cpp#L651-L653